snippets/rust/src/receive_onchain.rs

```diff
@@ -65,7 +65,7 @@ async fn get_channel_opening_fees(sdk: Arc<BreezServices>, amount_msat: u64) ->
     // ANCHOR: get-channel-opening-fees
     let channel_fees = sdk
         .open_channel_fee(OpenChannelFeeRequest {
-            amount_msat,
+            amount_msat: Some(amount_msat),
```

Comment:
CI is complaining about this, probably because its pre-release. Maybe revert and we can add another PR to change the `open_channel_fee` request examples